apps/gp2-server/src/data-providers/working-group-network.data-provider.ts

```diff
@@ -0,0 +1,53 @@
+import { gp2 } from '@asap-hub/model';
+import { workingGroupNetworkRole } from '@asap-hub/model/src/gp2';
+import { SquidexGraphqlClient } from '@asap-hub/squidex';
+import {
+  FetchWorkingGroupNetworkQuery,
+  FetchWorkingGroupNetworkQueryVariables,
+} from '../autogenerated-gql/graphql';
+import { FETCH_WORKING_GROUP_NETWORK } from '../queries/working-group-network.queries';
+import { parseWorkingGroupToDataObject } from './working-group.data-provider';
+
+export interface WorkingGroupNetworkDataProvider {
+  fetch(): Promise<gp2.ListWorkingGroupNetworkDataObject>;
+}
+export class WorkingGroupNetworkSquidexDataProvider
+  implements WorkingGroupNetworkDataProvider
+{
+  constructor(private squidexGraphlClient: SquidexGraphqlClient) {}
+
+  async fetch(): Promise<gp2.ListWorkingGroupNetworkDataObject> {
+    const { queryWorkingGroupNetworkContents: networks } =
+      await this.squidexGraphlClient.request<
+        FetchWorkingGroupNetworkQuery,
+        FetchWorkingGroupNetworkQueryVariables
+      >(FETCH_WORKING_GROUP_NETWORK);
+
+    if (!networks || networks.length === 0 || !networks[0]) {
```

Comment:
surely this conditional would do the job without the length check?
```suggestion
    if (!networks || !networks[0]) {
```